File: CS/DXWebApplication3/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web.Mvc;
using DXWebApplication3.Models;

namespace DXWebApplication3.Controllers {
    public class HomeController : Controller {

        NWindEntities db = new NWindEntities();
        
        public ActionResult Index() {
            return View();
        }

        [ValidateInput(false)]
        public ActionResult PivotGridPartial() {
            var model = db.ProductReports;
            return PartialView("_PivotGridPartial", model.ToList());
        }
    }
}

File: VB/DXWebApplication3/Controllers/HomeController.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web.Mvc
Imports DXWebApplication3.Models

Namespace DXWebApplication3.Controllers
    Public Class HomeController
        Inherits Controller

        Private db As New NWindEntities()

        Public Function Index() As ActionResult
            Return View()
        End Function

        <ValidateInput(False)>
        Public Function PivotGridPartial() As ActionResult
            Dim model = db.ProductReports
            Return PartialView("_PivotGridPartial", model.ToList())
        End Function
    End Class
End Namespace